lab11_Mvc/Models/PersonOfTheYear.cs

```diff
@@ -16,5 +19,35 @@ public class PersonOfTheYear
 		public string Title { get; set; }
 		public string Category { get; set; }
 		public string Context { get; set; }
+
+
+		public static List<PersonOfTheYear> GetStaff(int selectedStartYear, int selectedEndYear)
+		{
+			string[] PersonOfTheYear = File.ReadAllLines("App_Data/personOfTheYear.csv");
+
+			return PersonOfTheYear
+			   .Skip(1) // skip the header
+			   .From(year in PersonOfTheYear)
+			   .where(selectedStartYear<= year && selectedEndYear>=year)
+			   .Select(personLine => personLine.Split(",")) // split line into cells
+			   .Select(personCells => new PersonOfTheYear // make new Staff from cells
+			   {
+				    
+				   Year= int.Parse(personCells[0]),
+				   Honor= personCells[1],
+				   Name= personCells[2],
+				   Country= personCells[3],
+				   BirthYear= int.Parse(personCells[4]), 
+				   DeathYear= int.Parse(personCells[5]),
```

Comment:
This won't work for all values! Two options:

1. `BirthYear = personCells[4] == "" ? 0 : int.Parse(personCells[4]),`
2. `BirthYear = int.TryParse(personCells[4], out int birthYear) ? birthYear : 0,`